token-lending/program/src/processor.rs

```diff
@@ -427,7 +427,12 @@ fn process_refresh_reserve(program_id: &Pubkey, accounts: &[AccountInfo]) -> Pro
     let account_info_iter = &mut accounts.iter().peekable();
     let reserve_info = next_account_info(account_info_iter)?;
     let pyth_price_info = next_account_info(account_info_iter)?;
-    let switchboard_feed_info = next_account_info(account_info_iter)?;
+    // set switchboard to a placeholder account info
+    let mut switchboard_feed_info = pyth_price_info;
+    // if the next account info exists and is not the clock set it to be switchboard
+    if account_info_iter.peek().map(|a| a.key) != Some(&clock::ID) {
```

Comment:
what happens if there's no switchboard and no clock? seems like next_account_info() gets called and it'll panic (bc None != Some(clock::id))